### PR TITLE
Add Intl.getCanonicalLocales

### PR DESCRIPTION
**Pull request by zbraniecki**

### CODE REVIEW

File: src/9.negotiation.js [hunk 1]
```diff
@@ -20,6 +20,20 @@ import {
 
 const expUnicodeExSeq = /-u(?:-[0-9a-z]{2,8})+/gi; // See `extension` below
 
+defineProperty(Intl, 'getCanonicalLocales', {
```
Comment (caridy) [inline src/9.negotiation.js]: I don't think Intl is defined here. This probably belongs to src/intl.js

Comment (zbraniecki) [inline src/9.negotiation.js]: src/intl.js only defined Intl object. All `defineProperty` are places in respective files (like numberformat or datetimeformat).

Is it ok if I just do what those two files do - import Intl from src/intl.js?

Comment (caridy) [inline src/9.negotiation.js]: In the spec, it belongs to the Intl section, which was expanded to support new members.

Comment (caridy) [inline src/9.negotiation.js]: It should be the other way around from what you have in this PR.

**Pull request merged**